plugin/vim-printer.vim

```diff
@@ -22,6 +22,12 @@ let s:vim_printer_items_full = {
             \ 'sh': 'echo "{$}" "${$}"',
             \ 'bash': 'echo "{$}" "${$}"',
             \ 'zsh': 'echo "{$}" "${$}"'
+            \ 'java': 'System.out.println("{$}:" + {$});'
+            \ 'lua': 'print("{$}:", {$})'
+            \ 'swift': 'print("{$}:", {$})'
+            \ 'matlab': 'fprintf("{$}:%i\n", {$})'
```

Comment:
From the look of it `%i` seems to work only for integers. I don't much about matlab, but I found a function called `disp` which should let you display anything without knowing the type of that variable. Can we make use of that maybe?